first-step-spring-class-config/src/main/java/org/elearning/portfolio/user/UserDAOImpl.java

```diff
@@ -0,0 +1,79 @@
+package org.elearning.portfolio.user;
+
+
+import java.util.List;
+import javax.sql.DataSource;
+import javax.annotation.PostConstruct;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Repository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.elearning.portfolio.message.*;
+import org.elearning.portfolio.CreateDataSourceForJdbcTemplate;
+
+
+
+/**
+ * Created by rzonsol on 09.10.2016.
+ */
+
+public class UserDAOImpl implements UserDAO {
+
+    private DataSource dataSource;
+    private JdbcTemplate jdbcTemplateObject;
+
+
+    public User  getUser(Integer userId){
+        String sqlCom = "SELECT * FROM `USER` WHERE ID = ? ";
+        List <User> users = jdbcTemplateObject.query(sqlCom,
+                new Object[]{userId}, new UserMapper());
+
+        return users.get(0);
+    }
+
+    @Autowired
+    public void setDataSource(CreateDataSourceForJdbcTemplate dataSource) {
```

Comment:
Bardzo brzydka implementacja:
1. metoda nazywa się setDataSource a robisz coś na CreateDataSourceForJdbcTemplate 
2. parametr wejściowy nazywa się dataSource a nim właściwie nie jest
3. this.dataSource = dataSource.getDataSource(); - to też brzydko wygląda